core/contracts/contracts/IShield.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: CC0
+pragma solidity ^0.6.9;
+
+interface IShield {
```

Comment:
There should be documentation somewhere that lets people know where the source code resides for Shield.sol and Verifier.sol since those two contracts are missing from this directory

Reply:
Will address this 👍